copier.yml

```diff
@@ -14,4 +14,6 @@ min_python:
 max_python:
   type: str
   help: Maximum Python version that is known to work?
+standard_python: '3.10'
```

Comment:
> Cannot comment above but there is `python-version: '3.10'` on L19. Can we also have this as a parameter, e.g. `standard_python` or `default_python`?

@nvaytet  I made it as a parameter here based on the [comment](https://github.com/scipp/copier_template/pull/7/files#r1263427074). I think it's better to be hard-coded like but not entirely sure...